src/main.ts

```diff
@@ -31,7 +38,7 @@ function buildNpmDependencies(): any {
 }
 
 const command: Command = {
-	description: 'test your application',
+	description: 'this command will implicitly build your application and then run tests against that build',
```

Comment:
I already have a change for this in [a branch](https://github.com/bryanforbes/cli-test-intern/tree/fix-descriptions) I've created for fixing strings. Should we land mine first?